modules/ept-frontend/client/components/post_processing/post-processing.directive.js

```diff
@@ -89,11 +89,7 @@ module.exports = ['$timeout', '$filter', '$compile', function($timeout, $filter,
         var classString = styleString.replace('class="bbcode-', '');
         classString = classString.replace('"', '');
 
-        if (classString.indexOf('color-_') === 0) {
```

Comment:
I removed this if statement as `color-_` will never exist (because I removed that part in the parser). I apologize for this not being clear since I applied my own formatting on the `bbcode.js`-file.